SeaPublicWebsite.BusinessLogic/Models/PropertyData.cs

```diff
@@ -44,6 +44,56 @@ public class PropertyData
     public int? HoursOfHeatingMorning { get; set; }
     public int? HoursOfHeatingEvening { get; set; }
     public decimal? Temperature { get; set; }
+    public PropertyData Backup { get; set; }
 
     public List<PropertyRecommendation> PropertyRecommendations { get; set; }
+
+    public void CreateBackup()
```

Comment:
`Backup` is quite loaded term so I'd like to avoid it here. I think the best we've come up with so far is `UneditedData`